### PR TITLE
Teams Graph: Add chat app CRUD apis for beta

### PR DESCRIPTION
**Pull request by AkJo**
Adding CRUD APIs for apps in chats. 
Also, fixing inconsistencies and examples for app crud in team and personal scope.

### CODE REVIEW

File: api-reference/beta/api/chat-post-installedapps.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+### Response
+
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "microsoft.graph.teamsAppInstallation"
+} -->
+
+```http
```
Comment (Lauragra) [inline api-reference/beta/api/chat-post-installedapps.md:79]: The response example doesn't match the description of the response.

File: api-reference/beta/api/chat-upgrade-installedapps.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+### Response
+
+<!-- {
+  "blockType": "response",
+  "truncated": true
+}
+-->
+
+```http
+HTTP/1.1 204 No Content
```
Comment (Lauragra) [inline api-reference/beta/api/chat-upgrade-installedapps.md:61]: The response doesn't match the description in the Response section.

File: api-reference/beta/api/chat-upgrade-installedapps.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Update app in a chat"
```
Comment (Lauragra) [inline api-reference/beta/api/chat-upgrade-installedapps.md:2]: The file name should be teamsappinstallation-upgrade and the title should be teamAppInstallation: upgrade. It looks like it's an action, not an update CRUD operation.

File: api-reference/beta/api/chat-post-installedapps.md (hunk 1 above)
Comment (AkJo) [reply api-reference/beta/api/chat-post-installedapps.md:79]: Should be updated now.

File: api-reference/beta/api/chat-upgrade-installedapps.md (hunk 2 above)
Comment (AkJo) [reply api-reference/beta/api/chat-upgrade-installedapps.md:61]: This should be updated now.

File: api-reference/beta/api/chat-upgrade-installedapps.md (hunk 3 above)
Comment (AkJo) [reply api-reference/beta/api/chat-upgrade-installedapps.md:2]: This kind of comes back to: how do we name the files such that we can have separate files for /chats/id/installedApps/id/upgrade and /teams/id/installedApps/id/upgrade?

**Pull request merged**